### PR TITLE
[doc](multi-catalog)huawei mrs event sync doc

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

huawei mrs event sync doc

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

File: docs/zh-CN/docs/lakehouse/multi-catalog/multi-catalog.md [hunk 1]
```diff
@@ -365,6 +365,17 @@ Doris 的权限管理功能提供了对 Catalog 层级的扩展，具体可参
...
+华为的MRS需要更改hivemetastore-site.xml 并重启HMS和HiveServer2：
+
+```
+<property>
+    <name>metastore.transactional.event.listeners</name>
+    <value>org.apache.hive.hcatalog.listener.DbNotificationListener</value>
+</property>
+```
+
+注意：value是在原有值的基础上以逗号分隔追加，而不是覆盖。
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/multi-catalog.md:377]: Give an example

**Comment by zddr** [comment]
run buildall

**Review by zy-kkk — APPROVED** [review]
LGTM

**Pull request merged**